imports/startup/server/data.coffee

```diff
@@ -0,0 +1,164 @@
+{ prefixes, makeRequest, castBinding } = require './api.coffee'
+{ articles, agents } = require '../../data/collections.coffee'
+
+
+Meteor.publish 'articles', (amountToShow) ->
+  articles.find(
+    {},
+    {
+      limit: amountToShow
+      sort: { order: 1 }
+    }
+  )
+
+Meteor.publish 'recentAgentsForArticle', (articleId) ->
+  agents.find({articleId: articleId})
+
+
+order = 0
+
+# Remove all existing items on startup
+articles.remove({})
+agents.remove({})
+
+console.log "Populating empty collections with data from SPARQL..."
+query = prefixes + """
```

Comment:
I've added a function where the only argument is the LIMIT's value. That will make the script initialize that string object upon every call, but it's a relativly small price to pay for having less LOCs.